juniper_official/Kernel/check-stuck-ifstate-clients-iagent.rule

```diff
@@ -0,0 +1,47 @@
+/*
+ * This rule checks if there are any stuck ifstate clients present in the system.
+ * Though rare, but when any Ifstate client (application/FPC peer) gets stuck,
+ * it can result in resource accumulation and eventually can lead to traffic loss.
+ *
+ * It checks if the "slow peers" alarm has been raised on the device.
+ */
+iceberg {
+    topic kernel.ifstate {
+        rule check-stuck-ifstate-clients-iagent {
+            sensor stuck-client-sensor {
+                iAgent {
+                    file fetch_stuck_client.yml;
+                    table stuckclientTable;
+                    frequency 1m;
+                }
+            }
+            field stuck-client-info {
+                sensor stuck-client-sensor {
+                    path stuck_client_info;
+                }
+                type string;
+            }
+            trigger report-stuck-ifstate-clients-iagent {
```

Comment:
1) Provide trigger description and synopsis detailing sensor info.
2) Trigger frequency is must as default frequency is 10s and trigger gets evaluated once in 10s. but here sensor frequecy is 10 mins hence trigger frequency can be greaterthan equal to 10 mins.
e.g. synopsis "Routing engine GRES kpi";
       description "sets health based on Routing engine CPU utilization for 15 minutes";

Reply:
Addressed.